jobsdb/jobsdb.go

```diff
@@ -188,39 +188,39 @@ type JobsDB interface {
 	WithStoreSafeTx(func(tx StoreSafeTx) error) error
 
 	// Store stores the provided jobs to the database
-	Store(jobList []*JobT) error
+	Store(ctx context.Context, jobList []*JobT) error
 
 	// StoreInTx stores the provided jobs to the database using an existing transaction.
 	// Please ensure that you are using an StoreSafeTx, e.g.
 	//    jobsdb.WithStoreSafeTx(func(tx StoreSafeTx) error {
 	//	      jobsdb.StoreInTx(tx, jobList)
```

Comment:
Need to update the example in the documentation as well. Same thing applies for all docs below
```suggestion
	//	      jobsdb.StoreInTx(ctx, tx, jobList)
```